libs/api/domains/notifications/src/lib/notifications.module.ts

```diff
@@ -0,0 +1,23 @@
+import { AuthModule } from '@island.is/auth-nest-tools'
+import { FeatureFlagModule } from '@island.is/nest/feature-flags'
+import { logger, LOGGER_PROVIDER } from '@island.is/logging'
+import { UserNotificationClientModule } from '@island.is/clients/user-notification'
+
+import { Module } from '@nestjs/common'
+
+import { NotificationsResolver } from './notifications.resolver'
+import { NotificationsService } from './notifications.service'
+
+@Module({
+  imports: [AuthModule, FeatureFlagModule, UserNotificationClientModule],
```

Comment:
`AuthModule` isn't necessary here, as the root api app module loads it.
Looks like the `FeatureFlagModule` is unused?